lib/init/grass.py

```diff
@@ -1854,8 +1856,12 @@ def sh_like_startup(location, location_name, grass_env_file, sh):
     f = open(shell_rc_file, 'w')
 
     if sh == 'zsh':
+        # zsh does not have an unlimited history setting, so 1e8 is set as a proxy  
         if not os.getenv('SAVEHIST'):
-            os.environ['SAVEHIST'] = os.getenv('HISTSIZE')
+            os.environ['SAVEHIST'] = "100000000"
+            os.environ['HISTSIZE'] = os.getenv('SAVEHIST')
+            
+    
```

Comment:
It is not visible on GitHub, but these empty lines contain white space (visible e.g. in the command line). @cwhite911 You can probably remove both lines. It is readable without them too (plus only one line should be there according to PEP8). Check all lines you edited and remove trailing spaces if needed (again PEP8).